components/ButtonLink/ButtonLink.js

```diff
@@ -0,0 +1,13 @@
+import Link from 'next/link';
+
+export const ButtonLink = (props) => (
```

Comment:
It looks like a button, but it is actually a link. So that is why I used "ButtonLink" :D It is usually used for the "read more" link, it should be an UI component. I will change the name :)